app/src/main/java/com/connorlay/knitgrid/fragments/PatternDetailFragment.java

```diff
@@ -41,6 +46,8 @@
 
     private PatternPresenter mPatternPresenter;
 
+    private RowColumnListener mListener;
+
     public static PatternDetailFragment newInstance(PatternPresenter patternPresenter) {
```

Comment:
Now that I think about it, we should probably be injecting a listener both for viewing and editing of patterns. Both can implement the `RowColumnListener` so there is only one constructor. It seems weird to me that we are only injecting creation behavior, but not highlighting behavior :/